docs/development/building.md

```diff
@@ -1,36 +1,42 @@
-# Akri Build
+# Building
+
 Building Akri, whether locally or in the automated CI builds, leverages the same set of `make` commands.
 
-In essence, Akri components can be thought of as:
-1. Runtime components
-    1. Rust code: containers based on Rust code are built using `Cargo cross` and subsequent `docker build` commands include the cross-built binaries.
-        > Note: For Rust code, build/Dockerfile.* does NOT run `cargo build`, instead they simply copy cross-built binaries into the container
-    2. Other code: these containers can be .NET or python or whatever else ... the build/Dockerfile.* must do whatever building is required.
-2. Intermediate components: these containers are used as part of the build process and are not used in production explicitly
+In essence, Akri components can be thought of as: 1. Runtime components 1. Rust code: containers based on Rust code are built using `Cargo cross` and subsequent `docker build` commands include the cross-built binaries.
+
+{% hint style="info" %}
```

Comment:
what does this `{% hint ... %}` look like in an md viewer?  when i click the preview button in github, this file doesn't format well in markdown.